picard/album.py

```diff
@@ -288,19 +297,21 @@ def _release_request_finished(self, document, http, error):
                         error = False
             else:
                 try:
-                    parsed = self._parse_release(document)
+                    parse_result = self._parse_release(document)
                     config = get_config()
-                    if not parsed and config.setting['track_ars']:
+                    if parse_result == ParseResult.MISSING_TRACK_RELS:
```

Comment:
Isn't `config.setting['track_ars']` test still needed?

Reply:
It's already handled in `_parse_release`. The state `ParseResult.MISSING_TRACK_RELS` will only be set if this config setting is used.

That is another nice effect of introducing this enum, it simplifies the code here. The double check for the config setting was only done because of the ambiguity of the boolean return value.

Reply:
Thanks for the explanation.